php-fpm-healthcheck

```diff
@@ -103,10 +103,14 @@ check_fpm_health() {
     done
 }
 
+set +e
+
 GETOPT=$(getopt -o v --long verbose,accepted-conn:,listen-queue-len:,active-processes:,slow-requests: -n 'php-fpm-healthcheck' -- "$@")
 
 if test "$?" != 0; then >&2 echo "Invalid options, terminating." ; exit 3 ; fi
 
+set -e
```

Comment:
This is probably better done as `|| true` after the command that may fail, so it's clear which command's exit status we're ignoring.

Reply:
fixed!